3-networks/envs/dev/commons.auto.tfvars

```diff
@@ -0,0 +1 @@
+../../commons.auto.tfvars
```

Comment:
In step 4-projects, we call this file `common.auto.example.tfvars` instead of `commons.auto.example.tfvars`
Can we keep that standard? - see https://github.com/terraform-google-modules/terraform-example-foundation/tree/develop/4-projects

Reply:
Fixed!